newVersion.py

```diff
@@ -552,8 +585,14 @@ def get_first_entry_of_last_line(results):
         return entries[0]
 
 def get_last_commit_id(dir, tag_filter):
-    results = subprocess.check_output(['git', 'show-ref', '--tags', tag_filter], cwd=dir, encoding='UTF-8')
-    return get_first_entry_of_last_line(results)
+    try:
+        results = subprocess.check_output(['git', 'show-ref', '--tags', tag_filter], cwd=dir, encoding='UTF-8')
+        return get_first_entry_of_last_line(results)
+    except Exception as e:
+        if dir == sdk_dirs.sdk_ios:
+            # On the first 0.20 release, we need to use the ID from the last 0.10 release.
+            return "3135ae431fb9db9c45fe91739dd98044ab257a92"
+        raise(e)
```

Comment:
Is there a way we can detect a change in minor version and do this automatically rather than hardcoding it?

Reply:
Now that 0.20.1 is released, this code cannot be reached. It was a shim just to get the one release out.

Reply:
I'm sure that it could have been "smarter", but I didn't consider it to be worth it for one release.

Reply:
OK, will it happen again when we go to 0.3?

Reply:
Probably, so long term a real fix is probably good. But I was in a rush to get 0.20.1 released, and doing it now won't work without actually changing the new version number (which will hopefully be 1.0, not 0.3).

Reply:
That's fine, I suggest we remove this code and leave a TODO comment.

Reply:
Done